tools/oneclickapps/prepare_digital_ocean_one_click_app_release.py

```diff
@@ -122,7 +122,7 @@ def setup_one_click_app_installer(droplet: digitalocean.Droplet) -> None:
 def send_message(content: str) -> None:
     request = {
         "type": "stream",
-        "to": os.environ["ZULIP_STREAM_TO_SEND_BOT_MESSAGES"],
+        "to": "kandra ops",
         "topic": "DigitalOcean One Click App",
```

Comment:
I think probably "digitalocean installer" is probably a more standard format topic :).